From 5dcea63213712c3ffdc56d4fc3d463955117b828 Mon Sep 17 00:00:00 2001
From: Ain <41307858+nero@users.noreply.github.com>
Date: Thu, 9 Aug 2018 21:38:09 +0200
Subject: [PATCH 4/5] Fix installation path customisation

Previously, datadir was ignored when installing, using the hardcoded
default value.
---
 Makefile | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 70a48a4..0fa7efd 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,7 @@
 
 VERSION		:= 3.3.0
 
+sbindir		?= /sbin
 sysconfdir	?= /etc/mkinitfs
 datarootdir	?= /usr/share
 datadir		?= $(datarootdir)/mkinitfs
@@ -116,13 +117,13 @@ nlplug-findfs: nlplug-findfs.o
 
 install: $(SBIN_FILES) $(SHARE_FILES) $(CONF_FILES)
 	for i in $(SBIN_FILES); do \
-		$(INSTALL) -Dm755 $$i $(DESTDIR)/sbin/$$i;\
+		$(INSTALL) -Dm755 $$i $(DESTDIR)/$(sbindir)/$$i;\
 	done
 	for i in $(CONF_FILES); do \
-		$(INSTALL) -Dm644 $$i $(DESTDIR)/etc/mkinitfs/$$i;\
+		$(INSTALL) -Dm644 $$i $(DESTDIR)/$(sysconfdir)/$$i;\
 	done
 	for i in $(SHARE_FILES); do \
-		$(INSTALL) -D $$i $(DESTDIR)/usr/share/mkinitfs/$$i;\
+		$(INSTALL) -D $$i $(DESTDIR)/$(datadir)/$$i;\
 	done
 	for i in $(MAN_FILES); do \
 		$(INSTALL) -D $$i $(DESTDIR)$(mandir)/man$${i##*.}/$$i;\
-- 
2.18.0

